app/screens/channel/channel.js

```diff
@@ -257,9 +259,15 @@ export default class Channel extends PureComponent {
 
             const Loading = require('app/components/channel_loader').default;
             return (
-                <View style={style.loading}>
-                    <Loading channelIsLoading={true}/>
-                </View>
+                <SafeAreaView navigator={navigator}>
+                    <View style={style.loading}>
+                        <EmptyToolbar
+                            theme={theme}
+                            isLandscape={this.props.isLandscape}
+                        />
+                        <Loading channelIsLoading={true}/>
```

Comment:
Nit: You can avoid true here `<Loading channelIsLoading />`. It is just a convention though as long as we stick with the same format it works. We can enforce a eslint rule for the same

Reply:
I don't know about others, but my personal preference is to be explicit here, and I do think we have an eslint rule to be explicit

Reply:
I can live with that 👍